src/components/Footer.astro

```diff
@@ -34,4 +34,27 @@ import Twitter from "../icons/twitter.astro";
   #socials a:hover {
     @apply fill-current text-white;
   }
-</style>
\ No newline at end of file
+
+  body {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+
+</style>
+
+
+<script>
+  $(document).click(function() {
+    $("#logo").effect("shake", {direction:"left", times:3, distance:400}, 400);
+  });
+
+  $("#alcoholic").click(function() {
+    // $("#alcoholic").effect("shake");
```

Comment:
nit: remove dead code

Reply:
nit or not it must be fixed . . . . push on the way